jinja/index.html

```diff
@@ -14,6 +14,32 @@
         <div class="row text-center">
             <h3 class="col-12 mt-5 mb-5 grey-text" style="font-family: 'Roboto', sans-serif !important;">A cross-platform Python library for differentiable programming of quantum computers. Train a quantum computer the same way as a neural network.</h3>
         </div>
+
+
+        <div class="row pb-5 align-items-center justify-content-center">
+            <div class="m-3 card-image">
+                <a href="https://github.com/PennyLaneAI/pennylane/blob/master/unitaryhack.md">
+                    <img width="200px" src="https://unitaryfund.github.io/unitaryhack/assets/logo-date.png" />
+                </a>
+            </div>
+            <div class="m-3 text-center align-items-center">
+                <p class="black-text">
+                    <strong>PennyLane is taking part in unitaryHACK.</strong>
```

Comment:
Could we have this part in a larger font size? Maybe 1.5-2x the current height

Reply:
The same font as in the headers of the cards below ("Learn", "Play", "Hack") might do the trick

Reply:
Sure @co9olguy will do those changes.